Add sushi-chef for global youth communities

dkim0526 opened the pull request:
##### Description
This PR is to add sushi chef for global-youth-communities to import content from `global-youth-communities` into Kolibri studio. 

- update in sushichef.py
- add a json file that contains content information
- add a thumbnail image
- update some readme content.

dkim0526 committed:
First pass at sushi-chef-global-communities

Add first pass changes for suchi-chef-global.
Currently, channel description is too long,
which throws an error. For now, use None.

- Add .DS_Store in gitignore
- Add pypdf2 in requirement.txt
- Add smaller downloader.py in util dir
- Add pdf structure in page_structure.json file
- Update README content

dkim0526 committed:
Add a thumbnail and loggings for handling errors

Add a thumbnail image from twitter website
Add some logging for handling errors

jayoshih commented on sushichef.py:
```diff
@@ -45,31 +53,82 @@ class MyChef(SushiChef):
+        page_contents = downloader.read("{url}".format(url=url))
+    except Exception as e:
+        LOGGER.error("   Error on downloading pdf from {url} : {e}".format(url=url, e=e))
+        sys.exit(1)
```
Might be good to keep going to get as much content as possible

jayoshih commented on sushichef.py:
```diff
@@ -45,31 +53,82 @@ class MyChef(SushiChef):
+
+    if pdf_size < page_start-1 or pdf_size < page_end:
+        LOGGER.error("   Error with invalid page information on {} with page {} and page {}.".format(title, page_start, page_end))
+        sys.exit(1)
```
Just continue if an error is thrown (errors will be shown on the sushibar)

jayoshih reviewed (changes requested):
Nice work! Just a couple changes

dkim0526 committed:
Delete sys.exit to record more info on the sushibar

Instead of stopping with sys.exit call when there
is an error, continue running so that sushibar
will have more information to check the record

- Delete sys.exit call to record more info on the sushibar
